.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
Maybe we can start tagging our releases with the runtime version that we'll need to update on every change?